src/dispatch/plugins/dispatch_slack/plugin.py

```diff
@@ -198,20 +198,23 @@ def __init__(self):
 
     def get(self, email: str, **kwargs):
         """Fetch user info by email."""
+        team = department = weblink = ""
+
         profile = get_user_profile_by_email(self.client, email)
+        profile_fields = profile.get("fields")
+        if profile_fields:
+            team = profile_fields.get(SLACK_PROFILE_TEAM_FIELD_ID, {}).get("value", "")
```

Comment:
This type of think this has come up in a couple of different places, is empty string good or is something more explicit like "Unknown" better?

Reply:
Let's make it explicit here like we do in the participant and individual service modules.